network/init-neutrond.sh

```diff
@@ -825,10 +825,11 @@ set_genesis_param max_gas                               "\"1000000000\""
 set_genesis_param vote_extensions_enable_height         "\"1\""                                           # consensus_params
 set_genesis_param_jq ".app_state.marketmap.params.admin" "\"$ADMIN_MODULE_ADDRESS\""                      # marketmap
 set_genesis_param_jq ".app_state.marketmap.params.market_authorities" "[\"$ADMIN_MODULE_ADDRESS\"]"       # marketmap
+set_genesis_param_jq ".app_state.feemarket.params.max_block_utilization"    "\"1000000000\""              # feemarket
```

Comment:
I suggest to make this value a variable and use to set `max_gas` params and `max_block_utilization` param